ripe/atlas/tools/settings/__init__.py

```diff
@@ -241,4 +241,31 @@ def write(config):
             rc.write(payload)
 
 
+class Aliases(Configuration):
```

Comment:
I'm not thrilled about Aliases extending Configuration like this, as it's not a *kind* of configuration, they simply share properties etc.  A better way to do this would be to have them both inherit from a common parent that has just the features that both classes need.  As it is now, you'd expect aliases to be a sort of "configuration+" which it isn't.

Additionally, the name is plural which is generally not cool for a class name, but if it were turned into an actual singleton, then that might be ok.